libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -55,12 +87,29 @@ public static Task<IEnumerable<TeamsChannelAccount>> GetMembersAsync(ITurnContex
             }
         }
 
+        /// <summary>
+        /// Gets a paginated list of members of a team.
+        /// </summary>
+        /// <param name="turnContext"> Turn context. </param>
+        /// <param name="teamId"> ID of the Teams team. </param>
+        /// <param name="continuationToken"> continuationToken token. </param>
+        /// <param name="pageSize"> number of entries on the page. </param>
+        /// /// <param name="cancellationToken"> cancellation token. </param>
+        /// <returns>TeamsPagedMembersResult.</returns>
         public static Task<TeamsPagedMembersResult> GetPagedTeamMembersAsync(ITurnContext turnContext, string teamId = null, string continuationToken = default(string), int? pageSize = default(int?), CancellationToken cancellationToken = default)
         {
             var t = teamId ?? turnContext.Activity.TeamsGetTeamInfo()?.Id ?? throw new InvalidOperationException("This method is only valid within the scope of MS Teams Team.");
             return GetPagedMembersAsync(GetConnectorClient(turnContext), t, continuationToken, cancellationToken, pageSize);
         }
 
+        /// <summary>
+        /// Gets a pagined list of members of a non-team conversation.
```

Comment:
Is this right? Can you add more information about the usage of this method?

Reply:
updated.